FlashDevelop/Settings/Accessors.cs

```diff
@@ -294,7 +294,11 @@ public Int32 CaretWidth
         public Int32 ScrollWidth
         {
             get { return this.scrollWidth; }
-            set { this.scrollWidth = value; }
+            set
+            {
+                if (value < 0) throw new ArgumentOutOfRangeException();
```

Comment:
I don't think throwing error here is good. Maybe just reset it to 0 if its negative.